client/packages/common/src/ui/components/inputs/GenderInput/GenderInput.tsx

```diff
@@ -1,42 +1,52 @@
 import React from 'react';
-import { Select } from '@common/components';
-import { GenderType } from '@common/types';
-import { useTranslation } from '@common/intl';
-import { getGenderTranslationKey } from '@common/utils';
+import {
+  Select,
+  GenderType,
+  usePreference,
+  useTranslation,
+  getGenderTranslationKey,
+  PreferenceKey,
+} from '@openmsupply-client/common';
 
 type GenderInputProps = {
-  allowedValues?: GenderType[];
   value: GenderType | null | undefined;
   onChange: (value: GenderType) => void;
   disabled?: boolean;
   width?: string | number;
 };
 
 export const GenderInput = ({
-  // We'll make these the defaults for now, could be customised by country requirements later
-  allowedValues = [
-    GenderType.Female,
-    GenderType.Male,
-    GenderType.NonBinary,
-    GenderType.Unknown,
-  ],
   value,
   onChange,
   disabled,
   width = 250,
 }: GenderInputProps) => {
   const t = useTranslation();
+  const {
+    data: { genderOptions } = {
+      genderOptions: [
+        GenderType.Female,
+        GenderType.Male,
+        GenderType.NonBinary,
+        GenderType.Unknown,
+        GenderType.Transgender,
+        GenderType.TransgenderFemale,
+        GenderType.TransgenderMale,
```

Comment:
As noted above moved down here... not sure about `GenderType.Transgender` though...


![photo_2025-07-01 10 19 17](https://github.com/user-attachments/assets/e9bfe6a4-2eeb-49b1-bbea-1949d4f4c99d)

Reply:
@mark-prins would be good to review these options at some point?

Reply:
Female,
    Male,
    Transgender,
    TransgenderMale,
    TransgenderMaleHormone,
    TransgenderMaleSurgical,
    TransgenderFemale,
    TransgenderFemaleHormone,
    TransgenderFemaleSurgical,
    Unknown,
    NonBinary,
    
    Here are our options, is your proposal
    
    Female,
    Male,
    Transgender,
    TransgenderMale,
    TransgenderFemale,
    Other,
    NonBinary,
    PreferNotToSay
    
    ?